ldap/servers/slapd/slap.h

```diff
@@ -2345,6 +2346,7 @@ typedef struct _slapdEntryPoints
 #define CONFIG_LISTEN_BACKLOG_SIZE "nsslapd-listen-backlog-size"
 #define CONFIG_DYNAMIC_PLUGINS "nsslapd-dynamic-plugins"
 #define CONFIG_RETURN_DEFAULT_OPATTR "nsslapd-return-default-opattr"
+#define CONFIG_RETURN_ENTRY_DN "nsslapd-return-original-entrydn"
```

Comment:
Oh, and could you please also add a simple test case for this setting?

Reply:
done!